clients/cli/src/test/java/org/apache/gravitino/cli/TestMetalakeCommands.java

```diff
@@ -59,7 +59,9 @@ void testListMetalakesCommand() {
         spy(
             new GravitinoCommandLine(
                 mockCommandLine, mockOptions, CommandEntities.METALAKE, CommandActions.LIST));
-    doReturn(mockList).when(commandLine).newListMetalakes(GravitinoCommandLine.DEFAULT_URL, false);
+    doReturn(mockList)
+        .when(commandLine)
+        .newListMetalakes(GravitinoCommandLine.DEFAULT_URL, false, "");
```

Comment:
If --output is not specified on teh CLI then its value should be null not an empty string?

Reply:
DONE